frontend/src/js/external-forms/transformQueryToApi.ts

```diff
@@ -46,9 +46,16 @@ function transformFieldsToApi(fields, form) {
   }, {});
 }
 
+function setFormSubtype(formConfig: FormType) {
+  // Sets the subType member of the form request only if it was set in the form config.
+  // It might be better to introduce a metadata field in the config-type.ts that is hidden in the representation but is send to the backend.
```

Comment:
I think it might be better to use a single `type` field. 

The design of our API contract shouldn't be a result of workarounds for technical limitations in any one of the frameworks or libraries we use (backend OR frontend side).

This to me looks like a workaround

Reply:
Removes the frontend code changes